### PR TITLE
Allow for calling lift_fresh_copy manually

### PR DESCRIPTION
**Pull request by ezyang**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #113923

In this case, the input could be fake!  Just treat it normally in that case.

Fixes https://github.com/pytorch/pytorch/issues/113331

Signed-off-by: Edward Z. Yang <ezyang@meta.com>

cc @avikchaudhuri @gmagogsfm @zhxchen17 @tugsbayasgalan @angelayi @suo @ydwu4

### CODE REVIEW

**Review by eellison — APPROVED** [review]

**Review by bdhirsh — APPROVED** [review]

File: test/export/test_export.py [hunk 1]
```diff
@@ -1548,6 +1548,34 @@ def forward(self, x):
...
+
+        tensor_cpu = torch.randn(2, 4)
+        mask_cpu = torch.BoolTensor(
+            [[False,  True, False, False],
+            [False, False, False, False]]
+        )
+
+        m = M().eval()
+        # res_ref = m(tensor_cpu, mask_cpu)
+        # print("res_ref is: {}".format(res_ref), flush=True)
```
Comment (leslie-fang-intel) [inline test/export/test_export.py:1570]: Maybe delete these comments? Sorry, I left it in the example code in the issue.

Comment (ezyang) [reply test/export/test_export.py:1570]: I thought they were good comments 📦

**Review by leslie-fang-intel — APPROVED** [review]

**Pull request closed without merging**